cset-workflow/meta/rose-meta.conf

```
@@ -445,15 +446,25 @@ sort-key=cs3
 
 [template variables=CS_VARS]
 ns=Diagnostics
-description=List of all stash codes to extract.
-help=A list of stash codes to extract from the data, and compute the transect over.
-     It must contain pressure level data (i.e. be at least 3D with pressure,
-     latitude and longitude coordinates). In the future, this will also support
-     standard names to be LFRic compatible, along with other models.
+description=List of all variables to extract.
+help=A list of stash codes or variable names to extract from the data, and compute
+     the transect over. It must contain pressure level data (i.e. be at least 3D with pressure,
```

```suggestion
     the transect over. It must contain multi-level data (i.e. be at least 3D with pressure,
```
It could be another coordinate.